### ISSUE TITLE
Improve examples: Visible label is part of accessible name  [2ee8b8]

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1154]
The following inapplicable examples, while indeed not failures of 2.5.3, are still not a good practice. It'd be good if we came up with better examples than these:

```
<a aria-label="OK">Next</a>
```

```
<div role="tooltip" aria-label="OK">Next</div>
```

**Comment by Jym77** [comment]
Would `<a aria-label="WCAG">ACT rules</a>` be a good candidate?
